core/src/main/java/org/keycloak/crypto/integration/CryptoIntegration.java

```diff
@@ -0,0 +1,47 @@
+package org.keycloak.crypto.integration;
+
+import org.jboss.logging.Logger;
+import org.keycloak.common.util.BouncyIntegration;
+import org.keycloak.common.util.reflections.Reflections;
+
+/**
```

Comment:
Ideally this should be a KC SPI, and not something different, but for the time being we need this to work with adapters as well, but would be good to refactor this when we no longer have any adapters.

Reply:
Yes, exactly. I've considered Keycloak SPI from the beginning, but not possible due it needs to work with adapters as well.